sechub-adapter-pds/src/main/java/com/daimler/sechub/adapter/pds/PDSWebScanConfigImpl.java

```diff
@@ -82,6 +82,22 @@ public final PDSWebScanConfigBuilder setJobParameters(Map<String, String> jobPar
         @Override
         protected void customBuild(PDSWebScanConfigImpl config) {
             jobParameters.put(PDSDefaultParameterKeyConstants.PARAM_KEY_PDS_SCAN_TARGET_TYPE, config.getTargetType());
+            
+            // to get the base URl from the URL
+            // https://example.org/abc -> https://example.org
+            
+            try
```

Comment:
We have a dedicated source code format  - see https://github.com/mercedes-benz/sechub/wiki/Collaboration#source-formatter . It can be imported into eclipse . After this you can simply press `CTR` + `SHIFT` + `f` inside editor and your source is formatted as expected.